src/foam/dao/SequenceNumberDAO.js

```diff
@@ -94,6 +98,14 @@ foam.CLASS({
       javaType: 'foam.core.PropertyInfo',
       javaInfoType: 'foam.core.AbstractObjectPropertyInfo',
       javaFactory: 'return (foam.core.PropertyInfo)(getOf().getAxiomByName(getProperty()));'
+    },
+    {
+      class: 'Object',
+      name: 'uIDGenerator',
```

Comment:
@kgrgreer this seems odd.  Perhaps uidGenerator, but then we get getUidGenerator for a UIDGenerator.